src/app/shared/links/links-list/links-list-table/links-list-table.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="row">
+  <div class="col small text-right">
+    <strong>1&mdash;100</strong>&nbsp;of&nbsp;234
+    <a href="#"><i class="fas fa-angle-double-left"></i></a>
+    <a href="#"><i class="fas fa-angle-double-right"></i></a>
+  </div>
+</div>
+<div class="table-responsive">
+  <table class="table table-sm">
+    <thead>
+      <tr>
+        <th *ngIf="linkAttrCount > 0" [attr.colspan]="linkAttrCount" [title]="linkName">
+          <icons-presenter [icons]="linkIcons" [colors]="linkColors" [title]="linkName">
+          </icons-presenter>
+          {{linkName}}
+        </th>
+        <th [attr.colspan]="attributes.length - linkAttrCount" [title]="collectionName" class="{{linkAttrCount > 0 ? 'border-left' : ''}}">
+          <i [class]="collectionIcon" [style.color]="collectionColor"></i>&nbsp;{{collectionName}}
+        </th>
+      </tr>
+      <tr>
+        <th *ngFor="let attr of attributes; let j = index;" scope="col" title="{{attr}}" class="{{linkAttrCount == j && linkAttrCount > 0 ? 'border-left' : ''}}">{{attr}}</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let r of data; let idx = index"
+          class="clickable-row"
+          (click)="openLinked(idx)">
+        <td *ngFor="let attr of attributes; let i = index;"
+            title="{{r[attr]}}"
+            class="{{linkAttrCount == i && linkAttrCount > 0 ? 'border-left' : ''}} {{i < linkAttrCount ? 'always-white' : ''}}">
+          <input-box *ngIf="i < linkAttrCount"
```

Comment:
Consider using [class.border-left]="linkAttrCount === i && linkAttrCount > 0" instead. Same for the second one